### DISCUSSION TITLE
Prototype: IPCC Report and City Climate Change Plans

### DISCUSSION THREAD

**Discussion by mrchristian** [petermr/semanticClimate#32] [Ideas]
A prototype web service for easy knowledge access for use in climate plans, and supporting cultural and educational projects. 

The project would create short readers made from IPCC report for use by city climate change plan stakeholder and communities.

See description here: https://docs.google.com/document/d/1uhd1bhtsfYTjZ6g3YQTw6nvECQ_RwezBt_svlKFP5FE/edit#heading=h.t3bn9bwjwh70

**Comment by mrchristian** [comment]
Specs: Proof of concept prototye.

The idea is to being together two pipelines: py4ami and ADA. py4ami does the IPCC report search and retreival; and, ADA does the mult-format rendering.

Known challenges or questions:

- How do we perform a IPCC report search?
- Can the search retrieve the full-text of a search and deposit them in a repository directory as a single file per article (text excerpt)?
- Can each full-text file have a title
- How to convert the full-text files to Quarto's Markdown version that has a suffix of .qmd

Diagram for idea of workflow.

https://drive.google.com/file/d/19nr4ragNFhnV2_CeSPR9qnTObdlDlU_k/view?usp=sharing

![city-climate-plan drawio](https://github.com/petermr/semanticClimate/assets/772579/cc156eb6-ef19-4a72-914a-b3752f5a5109)

**Reply by mrchristian** [reply]
FYI this CSS typesetting engine should be something we consider in the future : Vivlioytle CLI - top quality typesetting from Japan and we can easily do layouts with CSS https://docs.vivliostyle.org/vivliostyle-cli.html

**Comment by petermr** [comment]
Thanks Simon, very useful

**Comment by petermr** [comment]
Cities will be one of the 3 focuses in our FSCI material (along with food and mountains)

Can we include the Indian City plans (Delhi and Mumbai) and also the 2023 newspaper report of Delhi climate (I think is in Slack).
Let's create a category:
# Related documents and news

**Comment by mrchristian** [comment]
Agenda

Meeting 12 July 2023

Meeting objective: To see if it is possible to make a proof of concept prototype IPCC Reader in the next two weeks before FSCI to be used by Hackathon Participants and by Publishing from Collections class?

- Introductions: People and projects (brief)
- FSCI timeline and beyond
- Workflow questions (See discussion: https://github.com/petermr/semanticClimate/discussions/32#discussioncomment-6375439
- What can we prepare in two weeks for FSCI?

**Comment by mrchristian** [comment]
Collecting Questions to use in a mockup for the City Climate Plan Reader prototype:

Questions will be collated in a Hedgedoc with discussion during the FSCI Hackathon happening in the Slack Channel.

1. Social justice and equity as well as mitigation strategies
2. Delhi climate plan questions
3.
